idea_town/frontend/templates/idea_town/frontend/index.html

```diff
@@ -3,6 +3,9 @@
 <head>
     <title>Idea Town</title>
     <link rel="stylesheet" href="{{ static('styles/main.css') }}">
+    <link rel="x-current-user" href="{{ url('users_self') }}">
+    {# TODO: Make this a dev / prod setting? Or, expose in an API endpoint? #}
```

Comment:
Hmm, we do have /api/ that lists links to experiments & details in the hypermedia-ish tradition. DRF doesn't list this particular resource. 

Might be worth landing this as-is and I can follow up to see about adding /api/users/self to the "table of contents". Can circle back with that to make /api/ the main thing linked in the head